RxCocoa/iOS/NSTextStorage+Rx.swift

```diff
@@ -11,6 +11,14 @@
     import RxSwift
 #endif
     import UIKit
+
+    extension NSTextStorage {
+        /// Use 'RxTextStorageDelegateProxy.extend' instead.
+        @available(*, unavailable, message: "createRxDelegateProxy is now unavailable, check DelegateProxyFactory")
+        public func createRxDelegateProxy() -> RxTextStorageDelegateProxy {
+            return RxTextStorageDelegateProxy(parentObject: self)
```

Comment:
I think we would need to implement all of these `unavailable` methods as `fatalError()`. I believe using unavailable methods result in compiler errors.

In the root of RxCocoa we have `Deprecated.swift` file. Could you please move all of these extensions to that file so they don't create deprecation noise :).